be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -538,8 +540,9 @@ void LakeTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkRequ
     if (_finish_mode == lake::kDontWriteTxnLog && request.eos() && (request.sender_id() == 0) &&
         response->status().status_code() == TStatusCode::OK) {
         rolk.unlock();
-        _txn_log_collector.wait();
-        auto st = _txn_log_collector.status();
+        auto t = request.timeout_ms() - (int64_t)(watch.elapsed_time() / 1000 / 1000);
```

Comment:
whether the underlying module can handle negative `t` properly?

Reply:
Yes, it can